client/src/pages/projects/[scpca_id].js

```diff
@@ -8,67 +8,72 @@ import { ProjectSamplesSummaryTable } from 'components/ProjectSamplesSummaryTabl
 import { Link } from 'components/Link'
 import { api } from 'api'
 import { useResponsive } from 'hooks/useResponsive'
+import Head from 'next/head'
 
 const Project = ({ project }) => {
   if (!project) return '404'
   const router = useRouter()
-
   const showSamples = router.asPath.indexOf('samples') !== -1
   const [activeIndex, setActiveIndex] = useState(showSamples ? 1 : 0)
   const onActive = (nextIndex) => setActiveIndex(nextIndex)
   const { responsive } = useResponsive()
   return (
-    <Box width="xlarge">
-      <ProjectHeader project={project} />
-      <Box pad={{ vertical: 'large' }}>
-        <Tabs activeIndex={activeIndex} onActive={onActive}>
-          <Tab title="Project Details">
-            <Box pad={{ vertical: 'large' }}>
-              <DetailsTable
-                data={project}
-                order={[
-                  'abstract',
-                  'disease_timings',
-                  'sample_count',
-                  'human_readable_pi_name',
-                  {
-                    label: 'Contact Information',
-                    value: (
-                      <Link
-                        label={`${project.contact_name} <${project.contact_email}>`}
-                        href={`mailto:${project.contact_email}`}
-                      />
-                    )
-                  }
-                ]}
-              />
-            </Box>
-            <Text size="large" weight="bold">
-              Sample Summary
-            </Text>
-            <Box
-              pad={{ vertical: 'medium' }}
-              width={{ max: 'full' }}
-              overflow="auto"
-            >
-              <ProjectSamplesSummaryTable summaries={project.summaries} />
-            </Box>
-          </Tab>
-          <Tab title="Sample Details">
-            <Box
-              pad={{ vertical: 'medium' }}
-              width={{ max: 'full' }}
-              overflow="auto"
-            >
-              <ProjectSamplesTable
-                project={project}
-                stickies={responsive(0, 3)}
-              />
-            </Box>
-          </Tab>
-        </Tabs>
+    <>
+      <Head>
```

Comment:
Instead of this can we do something like: `<PageTitle title={`project.title`} />` and just have the component render out that value with `- ScPCA Portal` as a suffix? 
So the logic would stay the same for the homepage etc (we should probably add a title for the about page).
Also, I believe you can remove the changes from `pages/projects/index.js` as the component in `_app.js` will result in the same title.

Reply:
Updated according to the feedback, thank you David!
(for the about page, currently its title is rendered as **About - ScPCA Portal,** pls let me know if you have any suggestions)

Reply:
All the changes have been reflected. Thank you for the feedback, David!